tests/integration/kubernetes/confidential_common.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+# Copyright 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+function setup_unencrypted_confidential_pod() {
+	get_pod_config_dir
+	
+	export SSH_KEY_FILE="${pod_config_dir}/confidential/unencrypted/ssh/unencrypted"
+	
+	# Use correct address in pod yaml
+	sed -i "s/image: image_addr/image: ghcr.io\/kata-containers\/test-images:unencrypted-${PR_NUMBER}/" "${pod_config_dir}/pod-confidential-unencrypted.yaml"
```

Comment:
IIRC as part of the CI pipeline the `ghcr.io/kata-containers/test-images:unencrypted-<PR>` image will be built and  pushed based on `tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/Dockerfile`. Do you have any thoughts on documentation so that users can run this locally as this won't work as is. I guess we'd want to update `image: image_addr` in the pod.yaml to point to an existing image and then this sed won't do anything, but there might be a better way?

Reply:
I've changed to using the unencrypted-nightly tag, and we keep a modified version of this sed just to make sure we test possible changes in the image as part of te PR.